zulipterminal/helper.py

```diff
@@ -173,7 +173,8 @@ def _set_count_in_view(controller: Any, new_count: int,
 
         msg_type = message['type']
         add_to_counts = True
-        if 'mentioned' in message['flags']:
+        if ('mentioned' in message['flags']
+                or 'wildcard_mentioned' in message['flags']):
```

Comment:
I'm merging a commit to master which will extend the line-length to 88 (ie. black's default), so likely your lines should simplify :)

It's not too bad with only two, but we could also consider a `set` approach here to avoid the duplication, like we've done recently in `soup2markup`.

Reply:
Ahh...The `set` approach seems better. I'll use that. :+1: